active/af1-dequeue-and-identify-phi.py

```diff
@@ -165,30 +165,30 @@ def annotate_clinical_notes(**kwargs):
             _log_failure_and_reqeue(message, "TEMP PERSON STORAGE", run_id, blobid, hdcpupdatedate)
             continue
 
-        to_review, skip_review = split_records_by_review_status(batch_records)
-        if _review_criterion(batch_records[blobid]):
```

Comment:
are we getting rid of this if statement entirely? that's the placeholder for optional/percent review right?

Reply:
or is it duplicated logic with the review,skip_review designation?

Reply:
it's duplicated logic. not necessary w/ line 168 above